spec/timezone.html

```diff
@@ -489,21 +500,31 @@ <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-formatisotimezoneoffsetstring" aoid="FormatISOTimeZoneOffsetString">
-      <h1>FormatISOTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
-      <p>
-        The abstract operation FormatISOTimeZoneOffsetString is similar to FormatTimeZoneOffsetString but rounds the offset to the nearest minute boundary, in order to produce a ±HH:MM format offset string for use in ISO 8601 strings.
-      </p>
+    <emu-clause id="sec-temporal-formattimezoneoffsetminutes" type="abstract operation">
+      <h1>
+        FormatTimeZoneOffsetMinutes (
+          _offsetMinutes_: an integer
+        ): a String
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It formats a time zone offset, in minutes, into a ±HH:MM format offset string for use in ISO 8601 strings or offset time zone identifiers.</dd>
+      </dl>
+      <emu-alg>
+        1. Return FormatTimeZoneOffsetNanoseconds(_offsetMinutes_ &times; (6 &times; 10<sup>10</sup>)).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-formatisotimezoneoffsetstring" type="abstract operation">
+      <h1>FormatTimeZoneOffsetRoundToMinutes ( _offsetNanoseconds_ )</h1>
```

Comment:
Suggestion: combine all operations that format a UTC offset (including particularly FormatTimeZoneOffsetMinutes and FormatTimeZoneOffsetRoundToMinutes) into a single operation.

Reply:
Makes sense, can do.

Reply:
I ended up reverting changes to formatting. So there's still two formatting methods: FormatTimeZoneOffsetString and FormatISOTimeZoneOffsetString. The only change I made was to make FormatISOTimeZoneOffsetString call FormatTimeZoneOffsetString to do the actual formatting. Otherwise no changes. Holler if this is a problem.